include/ola/Clock.h

```diff
@@ -214,21 +216,57 @@ class TimeStamp {
     explicit TimeStamp(const BaseTimeVal &time_val) : m_tv(time_val) {}
 };
 
-
 /**
  * @brief Used to get the current time.
  */
 class Clock {
  public:
   Clock() {}
   virtual ~Clock() {}
-  virtual void CurrentTime(TimeStamp *timestamp) const;
+  /**
+   * @brief Sets timestamp to the current monotonic time.
+   *
+   * The timestamp parameter will be set to the current monotonic time only if
+   * the system has a monotonic clock available.  If a monotonic clock is
+   * unavailable, this method will fallback to CurrentRealTime.
+   *
+   * The system's monotonic clock is not subject to discontinuous jumps due to
+   * administrative action, but may be affected by incremental adjustment due to
+   * time synchronization protocol e.g. NTP.  The monotonic clock does not
+   * advance while the system is suspended. An appropriate use of this method is
+   * to measure elapsed time.
+   * @sa Clock::CurrentRealTime
+   * @param timestamp A TimeStamp pointer
```

Comment:
This flags it as an output parameter in Doxygen land.
```suggestion
   * @param[out] timestamp A TimeStamp pointer
```

Reply:
Done.